app/controllers/completed_tasks_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CompletedTasksController < ApplicationController
+  before_action :authenticate_user!
+  def new
+  	@ct = CompletedTask.new
+  end
+  def create
+  	if(params[:is_checked].to_s == "true")
+  		current_user.completed_tasks.build(todo_item_id: params[:completed_task]["todo_item_id"])
+  	else
+  		current_user.completed_tasks.where(todo_item_id: params[:completed_task]["todo_item_id"]).destroy_all
```

Comment:
Line is too long. [105/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.